Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -46,8 +46,10 @@ @interface WPMediaPickerViewController ()
 
 @property (nonatomic, strong, readwrite) UISearchBar *searchBar;
 @property (nonatomic, strong) NSLayoutConstraint *searchBarTopConstraint;
+@property (nonatomic, assign) CGFloat currentKeyboardHeight;
 
 @property (nonatomic, strong) UIView *emptyView;
+@property (nonatomic, strong) UIView *containerEmptyView;
```

Comment:
I think this is more verbose
```suggestion
@property (nonatomic, strong) UIView *emptyViewContainer;
```

Reply:
Resolved here https://github.com/wordpress-mobile/MediaPicker-iOS/commit/e54f6a1a818e1226233fcbca6bb9649e57447b42 😄